src/v2/Apps/ViewingRoom/Components/ViewingRoomsLatestGrid.tsx

```diff
@@ -135,7 +140,18 @@ export const ViewingRoomsLatestGrid: React.FC<ViewingRoomsLatestGridProps> = pro
             const tag = getTagProps(status, distanceToOpen, distanceToClose)
 
             return (
-              <Link href={`/viewing-room/${slug}`} key={slug} noUnderline>
+              <Link
```

Comment:
This should be a `RouterLink` rather than a `Link`, so that we dno't do hard jumps between pages.